scripts/e2e_test/e2e_test.rb

```diff
@@ -86,6 +101,7 @@ def build_mailing
 
     Dir.chdir(CLI_ROOT) do
       system("npx yalc add")
+      system("npx yalc publish")
```

Comment:
@psugihara this is very necessary.  otherwise, it works while the version number is the same, but if you update the version number it does not get added to `~/.yalc/packages/mailing`
e.g.
```
ls ~/.yalc/packages/mailing
0.4.10		0.4.11-next.0	0.6.5		0.6.8
```